Alpaca.Markets/Interfaces/IAlpacaDataSubscription.cs

```diff
@@ -14,7 +14,7 @@ public interface IAlpacaDataSubscription
     /// Gets boolean flag indicating the current subscription status of this item.
     /// </summary>
     [UsedImplicitly]
-    Boolean Subscribed { get; }
+    long? Subscribed { get; }
```

Comment:
I'm not sure why do you need the exact time of receiving the subscription message but I suggest you re-design this PR in a more clear way. Just add the `OnSubscribedChanged` event into this interface and raise this event in the `Subscribed` property setter. Of course in this case you have to convert this auto-property into the 'classic' property with the backing field but this change will be backward compatible. Maybe you'll receive some error messages about incompatible API from the package validation analyzer but in this case, it's OK and I'll tell you how to solve them.

Reply:
as a general principle, timestamp carries more information. for example, whether the connection was dropped at some point

Reply:
1. Server never sends this timestamp in any message so the best value that you can have is a time when you've completed initial message processing in the SDK. I'm not sure it's useful for most users - just simple on/off state looks more appropriate.
2. If you want to track the time of subscription confirmation message receiving you can implement the approach that I've suggested. I'm OK with such an additional event in this place because in most cases it will be empty and will not decrease the overall performance significantly.

Reply:
got it, never mind the PR.. can u please help me build the main thing.. and get it running 
cause your examples dont work